### PR TITLE
Feature/euparl

### LINKED ISSUE #1250: European Parliament
All speeches of the European Parliament are archived, translated to English. Potentially, this will be added as a corpus in 2024.

### PR DESCRIPTION
**Pull request by BeritJanssen**
Add a corpus definition to add the (translated to) English European Parliament speeches, as provided by [Talk of Europe](https://ssh.datastations.nl/dataset.xhtml?persistentId=doi:10.17026/dans-x62-ew3m&version=1.0). Close #1250 .

### CODE REVIEW

File: backend/corpora/parliament/conftest.py
Comment (BeritJanssen) [inline backend/corpora/parliament/conftest.py:1]: Applied alphabetical sorting here.

File: backend/corpora/parliament/tests/test_import.py [hunk 1]
```diff
@@ -3,47 +3,49 @@
 import pytest
 from datetime import datetime
 
+from addcorpus.python_corpora.corpus import CorpusDefinition
 from addcorpus.python_corpora.load_corpus import load_corpus_definition
 
 CORPUS_TEST_DATA = [
```
Comment (JeltevanBoheemen) [inline backend/corpora/parliament/tests/test_import.py:9]: It takes a lot of scrolling to get to the actual tests in this file. I propose making `CORPUS_TEST_DATA` a fixture in a separate file.

File: backend/corpora/parliament/utils/field_defaults.py [hunk 2]
```diff
@@ -69,23 +69,23 @@ def country():
...
 
-def date():
+
+def date(min_date: datetime = MIN_DATE, max_date: datetime = MAX_DATE):
     "The date on which the debate took place."
     return FieldDefinition(
-        name='date',
-        display_name='Date',
-        description='The date on which the debate took place.',
+        name="date",
```
Comment (JeltevanBoheemen) [inline backend/corpora/parliament/utils/field_defaults.py:76]: Autoformat I suppose? The rest of the file is in single quotes. Don't really need to revert but pointing out for future reference.

File: backend/corpora/parliament/euparl.py [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+    ''' Parse all relevant metadata out of MembersOfParliament ttl to dict'''
+    speaker_dict = {}
+    speaker_graph = Graph()
+    speaker_graph.parse(filename)
+    speaker_subjects = speaker_graph.subjects(object=LPV.MemberOfParliament)
+    for speaker in speaker_subjects:
+        try:
+            name = speaker_graph.value(speaker, FOAF.name).value
+        except:
+            continue
```
Comment (JeltevanBoheemen) [inline backend/corpora/parliament/euparl.py:37]: Please add a comment on which cases are being ignored here, preferable with a more specific `Exception`

**Review by JeltevanBoheemen — CHANGES_REQUESTED** [review]
Code is clean. Usage of `rdflib` utilities makes it very readable. Two small requests.

**Pull request merged**